### PR TITLE
Update minimum_supported_agent_version in metadata.yaml.

### PR DESCRIPTION
**Pull request by stackdriver-instrumentation-release**

### CODE REVIEW

File: integration_test/third_party_apps_test/applications/dcgm/metadata.yaml [hunk 1]
```diff
@@ -74,8 +74,7 @@ expected_metrics:
       - *uuid_label
       - *gpu_number_label
     notes:
-      - &dcgm_profiling_note
-        Not available on GPU models P100 and P4.
+      - &dcgm_profiling_note Not available on GPU models P100 and P4.
```
Comment (igorpeshansky) [inline integration_test/third_party_apps_test/applications/dcgm/metadata.yaml:77]: Is this change intentional? If not, can we please keep the note on its own line?

Or is this forced by yamlfmt or something?

Comment (igorpeshansky) [reply integration_test/third_party_apps_test/applications/dcgm/metadata.yaml:77]: Never mind, this is the result of dumping out the yaml in a Python script. Let's just keep it this way, I guess.

**Review by igorpeshansky — APPROVED** [review]
This CL is a no-op modulo formatting changes, so LGTM :shipit:

**Review by LujieDuan — APPROVED** [review]
LGTM!
